### ISSUE TITLE
dynamo 0.8.1 installed, but I got this error message of dynamo crash.

### ISSUE THREAD

**Issue by Hyun-Tak** [DynamoDS/Dynamo#4471]
Hi, I installed DynamoInstall 0.8.1 20150416T0453.exe. 
But, sandbox version and dynamo on Revit 2014/2015 all crashed. I got this message as following;

Object reference not set to an instance of an object.

   at Dynamo.UI.Views.LibrarySearchView.OnTopResultTargetUpdated(Object sender, DataTransferEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.Data.BindingExpression.OnTargetUpdated(DependencyObject d, DependencyProperty dp)
   at System.Windows.Data.BindingExpression.TransferValue(Object newValue, Boolean isASubPropertyChange)
   at System.Windows.Data.BindingExpression.Activate(Object item)
   at System.Windows.Data.BindingExpression.AttachToContext(AttachAttempt attempt)
   at System.Windows.Data.BindingExpression.MS.Internal.Data.IDataBindEngineClient.AttachToContext(Boolean lastChance)
   at MS.Internal.Data.DataBindEngine.Task.Run(Boolean lastChance)
   at MS.Internal.Data.DataBindEngine.Run(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)

---

on another popup window, the message says :  
Error registering effect: Unable to find [sharpdx_direct3d11_1_effects_x64.dll] in the PATH

**Comment by jnealb** [comment]
@Hyun-Tak Do you have DirectX installed? If so, what version? You can run the Windows Dxdiag utility to find out. What version of Windows, and what language? Thanks.

**Comment by Hyun-Tak** [comment]
Hi, I have Windows 7 Enterprise 64 -bit, English, DirectX 11 installed in my Lenovo laptop.

**Comment by jnealb** [comment]
@Hyun-Tak We note you are using an older daily build - can you try with the latest?

**Comment by Hyun-Tak** [comment]
@jnealb  Thanks for your answer. But I installed the latest one which also is not working.  DynamoInstall0.8.1.20150513T0845.exe
I got this error message ;

Object reference not set to an instance of an object.

   at Dynamo.UI.Views.LibrarySearchView.OnTopResultTargetUpdated(Object sender, DataTransferEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.Data.BindingExpression.OnTargetUpdated(DependencyObject d, DependencyProperty dp)
   at System.Windows.Data.BindingExpression.TransferValue(Object newValue, Boolean isASubPropertyChange)
   at System.Windows.Data.BindingExpression.Activate(Object item)
   at System.Windows.Data.BindingExpression.AttachToContext(AttachAttempt attempt)
   at System.Windows.Data.BindingExpression.MS.Internal.Data.IDataBindEngineClient.AttachToContext(Boolean lastChance)
   at MS.Internal.Data.DataBindEngine.Task.Run(Boolean lastChance)
   at MS.Internal.Data.DataBindEngine.Run(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at MS.Internal.Threading.ExceptionFilterHelper.TryCatchWhen(Object source, Delegate method, Object args, Int32 numArgs, Delegate catchHandler)



Even DynamoSandbox is not working.
It has error messages like

can't start because D3DCOMPILER_44.dll is missing from your computer. - reinstalling the program.
Error registering effect: Unable to find [sharpdx_direct3d11_1_effects_x64.dll] in the PATH
the given key is not existent.

I think I need to reinstall the safe dynamo 0.8.0 until the next release version is out.
Thanks.
